client/components/Stack/StackBig.js

```diff
@@ -3,9 +3,8 @@ import {connect} from 'react-redux';
 import {Stack} from '../../components';
 import {
   flip3ForPlayer,
-  restartBigStackForPlayer
+  // restartBigStackForPlayer // being buggy
```

Comment:
I'm happy just getting rid of this if you guys are

Reply:
Sounds good to me.